index.js

```diff
@@ -43,7 +43,16 @@ var excludedKeys = {
 	$webkitStorageInfo: true,
 	$window: true
 };
+var isChromeExtension = (function () {
+	/* global chrome */
+	return (typeof chrome !== 'undefined') &&
+		(typeof chrome.runtime !== 'undefined') &&
+		(typeof chrome.runtime.id !== 'undefined');
+}());
 var hasAutomationEqualityBug = (function () {
+	if (isChromeExtension) {
+		return false;
```

Comment:
As I wrote in the issue, it is purely cosmetic issue (warnings printed to console.error, but no errors actually thrown out), it is just kind of annoying. I am not sure how to work around that.